django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,60 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.module_loading import import_string
+from django.utils.translation import ugettext as _
+
+
+def get_password_backends():
```

Comment:
I'm not sure about the `backend` terminology here. I think naming this function `get_password_validators` would be more consistent with the rest of the the code.